### ISSUE TITLE
Can you add jQuery v1.12.0 to the resource bundle?

### ISSUE THREAD

**Issue by Gitoffthelawn** [Synzvato/decentraleyes#83]
Thanks for the release of 1.3.5 :-)

Can you add jQuery v1.12.0 ?

This CDN is not being rerouted:
http://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js

**Comment by Synzvato** [comment]
My pleasure, I'm glad you like the update! The reason that jQuery v1.12.0 is not currently included is that, [according to W3Techs](http://w3techs.com/technologies/details/js-jquery/1.12.0/all), this particular release has a market share of just 0.2%.

**Comment by Gitoffthelawn** [comment]
Lol...the first site I happened to test the new version on uses that version of jQuery... what are the odds?

It's a major site (ranked 1311 worldwide and 495 in the USA as of May 7, 2016 [according to Alexa](http://www.alexa.com/siteinfo/drugs.com)): http://www.drugs.com/monograph/

I guess I just happened to hit the 0.2% right off the bat!  Or maybe the W3Tech methodology isn't quite as accurate or useful as they think!  Actually, after reading their methodology, it seems to have a serious flaw for this particular use: they don't weight sites at all.  In other words, the planet's most popular site has exactly the same weight as the 10 millionth most popular site.  Oops!

Given how tiny jQuery is, and how many sites call it from CDN's, I think you can probably justify going down to 0.2%, 0.1%, or even less.  At 0.1%, you're still talking about 1 out of every 1000 websites, which is still a huge number.  Even when you look at just the top 10 million sites, that's still 10,000 popular websites.

**Comment by Synzvato** [comment]
@Gitoffthelawn There will always be exceptions to (and flaws in) web crawling statistics. These metrics do result in a very high amount of counter hits for an add-on that would easily fit on a 1986 Perpendicular floppy disk. Focusing on individual websites is not workable and total coverage is not an option.

Speaking about the credibility of W3Techs, I should note that they provide a list of popular sites that depend on a specific version of a library (so they do know how to identify big players). When taking a look at the [requested version](http://w3techs.com/technologies/details/js-jquery/1.12.0/all), I see that ```drugs.com``` is indeed listed under "Popular sites".

By this logic, we should also include [v1.12.2 of jQuery](http://w3techs.com/technologies/details/js-jquery/1.12.2/all). Although it's used by less than less than 0.1% of all websites, ```gigazine.net``` _does_ use it and that site is ranked 743 worldwide, and 58 in Japan.

If you feel that W3Techs' statistics are completely off, feel free to supply different sources. If interested, feel free to take a look at #45 and continue this conversation under issue #7.

**Comment by Gitoffthelawn** [comment]
I think W3Tech's stats look good... I do think think their methodology is not ideal for this extension.  But I think it's sufficient to be a *very* solid foundation.

Each version of jQuery is ~50-90K uncompressed, with older versions being the smallest.  Firefox with a few extensions installed and a few tabs loaded is over 200MB, perhaps over 350MB.  As such, I think covering all sites in their "Popular sites" list makes sense.  What about you?

**Comment by Gitoffthelawn** [comment]
BTW, just a quick thought... if anyone has access to the raw W3Tech data, it would be easy to write a weighted formula to give more weight to more popular sites.  This would increase the hit/miss ratio.

**Comment by Synzvato** [comment]
@Gitoffthelawn I agree that covering all popular sites could be a good idea. Speaking of analyzing raw data and weighted formulas, you might want to take a peek at issue #45.

**Comment by Gitoffthelawn** [comment]
@Synzvato Thanks.  Just finished reading that (because you pointed me to it :+1: ).  Useful data.

**Comment by Gitoffthelawn** [comment]
@Synzvato Would you like me to create a new issue regarding covering libraries used by the most popular sites?  Or will this issue work for that?

**Comment by Synzvato** [comment]
@Gitoffthelawn If you'll be attempting to come up with a new way of analysing web crawling statistics, then feel free to create a new issue. We can then label it as an experiment.
